src/IECoreHoudini/SOP_SceneCacheSource.cpp

```diff
@@ -305,6 +308,79 @@ OP_ERROR SOP_SceneCacheSource::cookMySop( OP_Context &context )
 	return error();
 }
 
+ConstObjectPtr SOP_SceneCacheSource::buildTransformPointCloud( const IECoreScene::SceneInterface *scene, Imath::M44d transform, double time, Parameters &params, size_t rootSize, std::string currentPath, bool hasObject )
```

Comment:
`currentPath` appears to be unused/unnecessary. If it's desirable for some reason, perhaps `const string &` would be more appropriate. Perhaps `const &` would be more appropriate for `transform` too?